### DISCUSSION TITLE
Fix internal note links?

### DISCUSSION THREAD

**Discussion by bumper314** [vzhd1701/enex2notion#9] [Q&A] [answered]
Is this tool able to fix internal note links (links between notes)?  I assume not, because the .enex format is flawed in that it does not include the UUID/GUID of a note, on which the internal evernote:// link schema relies.

**Comment by vzhd1701** [comment, marked as answer]
No, currently it just skips over them. I could make it work with the help of my other tool [evernote-backup](https://github.com/vzhd1701/evernote-backup). It stores all user's notes. I could export notes' GUIDs into a "map" file and then use it with enex2notion to match corresponding notes with the links on upload. The problem is the links to external notes which are not owned by the user.

**Reply by bumper314** [reply]
Got it, and I expected as much.  I created this discussion to raise awareness of the issue for others, so you might add it to the "What is lost" section of the README.
